content/kubermatic/master/architecture/concept/kkp-concepts/resource-quotas/_index.en.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Resource Quotas"
+date = 2022-08-16T14:07:15+02:00
+weight = 20
+
++++
+
+## Resource Quotas in KKP
+
+Resource Quotas allow admins to set quotas on the amount of resources a subject can use. For now the only
+subject which is supported is Project, so the resource quotas currently limit the amount of resources that can be used project-wide.
+
+The resources in question are the resources of the user cluster:
+- CPU - the cumulated CPU used by the nodes on all clusters. 
+- Memory - the cumulated RAM used by the nodes on all clusters.
+- Storage - the cumulated disk size of the nodes on all clusters.
+
+This feature is available in the EE edition only.
+
+{{% notice note %}}
+**Note:** Do not confuse with the Resource Filter setting in the `Defaults And Limits` admin panel page. 
+That one just controls the size of the machines suggested to users in the KKP Dashboard during the cluster creation.
+{{% /notice %}}
+
+
+### Setting up the resource quotas
+
+The resource quotas are managed by admins either through the KKP UI or through the Resource Quota CRDs.
+
+Example ResourceQuota:
+```yaml
+apiVersion: kubermatic.k8c.io/v1
+kind: ResourceQuota
+metadata:
+  labels:
+    subject-kind: project
+    subject-name: tjqjkphnm6
+  name: project-tjqjkphnm6
+spec:
+  quota:
+    cpu: "100"
+    memory: 500G
+    storage: 350G
+  subject:
+    kind: project
+    name: tjqjkphnm6
+status:
+  globalUsage:
+    cpu: "2"
+    memory: 35G
+    storage: 127G
+  localUsage:
+    cpu: "1"
+    memory: 17G
+    storage: 100G
+```
+
+The quota fields use the [ResourceQuantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity) to 
+represent the values. One note is that CPU is denoted in single integer numbers.
+
+![Manage Quotas](/img/kubermatic/master/architecture/concepts/resource-quotas/quota-menu.png?classes=shadow,border "Manage Quotas")
+
+To simplify matters the UI uses GB as representation for Memory and Storage. The conversion from any value
+set in the ResourceQuota is done automatically by the API. 
+
+### Calculating the quota usage
+
+The ResourceQuota has 2 status fields:
+- `globalUsage` which shows the resource usage across all seeds
+- `localUsage` which shows the resource usage on the local seed
+
+Each seed cluster has a controller which calculates the `localUsage` by calculating the machine resource usage
```

Comment:
not sure if we shouldn't start with capital case (same as with Resource Quotas) as we are talking about kubernetes resource here.
```suggestion
Each seed cluster has a controller which calculates the `localUsage` by calculating the Machine resource usage
```